modules/adding-tls-termination.adoc

```diff
@@ -53,9 +55,9 @@ spec:
                 port:
                   number: 80
 ----
-<1> Specifies the name of an ingress.
-<2> The controller provisions the load balancer for this `Ingress` resource in a public subnet so that the load balancer is reachable over the internet.
-<3> The Amazon Resource Name of the certificate that you attach to the load balancer.
-<4> Defines the ingress class name.
+<1> Specifies the Ingress name.
+<2> The controller provisions the load balancer for Ingress in a public subnet to access load balancer over the internet.
```

Comment:
```suggestion
<2> The controller provisions the load balancer for Ingress in a public subnet to access the load balancer over the internet.
```
I am thinking maybe there should be a "the" before load balancer, but then you have another "the" before internet so I am not sure. Just a thought.